packages/mdx/test/compile.js

```diff
@@ -149,13 +149,14 @@ test('compile', async () => {
     '\nnote: the next deprecation is expected (preact is missing an export map)\n'
   )
 
-  assert.equal(
-    render(
-      h(await run(compileSync('?', {jsxImportSource: 'preact/compat'})), {})
-    ),
-    '<p>?</p>',
-    'should support an import source (`@jsxImportSource`)'
-  )
+  // To do: re-enable when `preact/compat` has a correct export map.
```

Comment:
Is there an open issue with preact to address this?
If so could it be noted in a comment?

Reply:
:thinking: it appears to have an export map https://github.com/preactjs/preact/blob/bd52611cade159a643b617794410f40a9d52eda3/package.json#L23
What issue is being encountered?

Reply:
The problem is that they don’t list `compat/jsx-runtime` in their export map

Reply:
reported